freqtrade/exchange/exchange.py

```diff
@@ -115,7 +115,7 @@ def __init__(self, config: Dict[str, Any], validate: bool = True) -> None:
 
         logger.info('Using Exchange "%s"', self.name)
 
-        if validate:
+        if validate and not exchange_config.get('skip_validation'):
```

Comment:
Personally, i would be more specific with this. Skipping all validations can be dangerous...

This will skip "all" validations - but at least for your usecase, only "validate_pairs" should be skipped - as stake-currency should still match (otherwise your backtests may show very odd results).

Reply:
good point, done